Kwf/Media/Output/ClearCacheInterface.php

```diff
@@ -0,0 +1,5 @@
+<?php
+interface Kwf_Media_Output_ClearCacheInterface
+{
+    public static function canCacheBeDeleted($id);
+}
```

Comment:
I think it's overkill to create a whole new interface only for one target component class. I would omit this interface and instead check directly for Kwc_Abstract_Image_Component in collectGarbage()